### PR TITLE
dockerfile: source youtube-dl from alpine repos

### LINKED ISSUE #191: youtube-dl taken down - future of podsync?
https://www.bleepingcomputer.com/news/software/youtube-dl-removed-from-github-after-riaa-dmca-notice/

I guess it likely Google will make a breaking change quite soon and the current version will stop working,

Are there alternatives?

### PR DESCRIPTION
**Pull request by jtagcat**
Does _not_ close #191, but a temporary solution.

WARN: updating youtube-dl will fail.
```
It looks like you installed youtube-dl with a package manager, pip, setup.py or a tarball. Please use that to update.
```
Instead, to update, use the cmd:
```
apk --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main add youtube-dl
```
I don't know how to do this in go, and if so, the binary needs a switch, if run from docker, indicating that it shall be upgraded via alpine's paman instead.

### CODE REVIEW

**Comment by jtagcat** [comment]
cc @mxpv for quick merge, build, and push of the image
thanks

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
 WORKDIR /app/
-RUN wget -O /usr/bin/youtube-dl https://github.com/ytdl-org/youtube-dl/releases/latest/download/youtube-dl && \
-    chmod +x /usr/bin/youtube-dl && \
-    apk --no-cache add ca-certificates python ffmpeg tzdata
+
+RUN apk --no-cache add ca-certificates python ffmpeg tzdata
+# see #191 for youtube-dl related questions
+RUN apk --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main add youtube-dl 
+# RUN wget -O /usr/bin/youtube-dl https://github.com/ytdl-org/youtube-dl/releases/latest/download/youtube-dl && \
```
Comment (mxpv) [inline Dockerfile:8]: Can you remove the commented line? We have VCS to remember what was there.

**Review by mxpv — COMMENTED** [review]
LGTM to a workaround until we figure out what to do with youtube-dl

**Comment by jtagcat** [comment]
@mxpv open a reverse PR?

**Pull request merged**